File: app/controllers/UserController.java
package controllers;

import modules.Environment;
import play.mvc.Controller;
import play.mvc.Result;
import views.html.profile;

/**
 * Created by Beemen on 03/08/2015.
 */
public class UserController extends Controller {
    public Result ProfileSummary() {
        return ok(profile.render(Environment.Current()));
    }
}

File: Controllers/UserController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace end_user_gui.Controllers
{
    public class UserController : Controller
    {
        public ActionResult ProfileSummary()
        {
            return View("profile", end_user_gui.Modules.Environment.Current());
        }
    }
}